src/plugins/discover/public/application/components/discover_legacy.tsx

```diff
@@ -0,0 +1,328 @@
+                aria-expanded={isSidebarClosed ? 'false' : 'true'}
+                aria-label="Toggle sidebar"
+                className="kuiCollapseButton kbnCollapsibleSidebar__collapseButton"
+              >
+                <span className={`kuiIcon ${sidebarCloseIcon}`} />
+              </button>
+            </div>
+            <div className={`dscWrapper ${mainSectionClassName}`}>
+              {resultState === 'none' && (
+                <DiscoverNoResults
+                  timeFieldName={opts.timefield}
+                  queryLanguage={state.query ? state.query.language : ''}
+                />
+              )}
+              {resultState === 'uninitialized' && <DiscoverUninitialized onRefresh={fetch} />}
+              {/* @TODO: Solved in the Angular way to satisfy functional test - should be improved*/}
+              <span style={{ display: resultState !== 'loading' ? 'none' : '' }}>
+                {fetchError && <DiscoverFetchError fetchError={fetchError} />}
+                <div className="dscOverlay" style={{ display: fetchError ? 'none' : '' }}>
+                  <LoadingSpinner />
+                </div>
+              </span>
+              {resultState === 'ready' && (
+                <div className="dscWrapper__content">
+                  <SkipBottomButton onClick={onSkipBottomButtonClick} />
+                  <HitsCounter
+                    hits={hits > 0 ? hits : 0}
+                    showResetButton={!!(savedSearch && savedSearch.id)}
+                    onResetQuery={resetQuery}
+                  />
+                  {opts.timefield && (
+                    <TimechartHeader
+                      dateFormat={opts.config.get('dateFormat')}
+                      timeRange={timeRange}
+                      options={search.aggs.intervalOptions}
+                      onChangeInterval={onChangeInterval}
+                      stateInterval={state.interval || ''}
+                      bucketInterval={bucketInterval}
+                    />
+                  )}
+
+                  {opts.timefield && (
+                    <section
+                      aria-label={i18n.translate('discover.histogramOfFoundDocumentsAriaLabel', {
+                        defaultMessage: 'Histogram of found documents',
+                      })}
+                      className="dscTimechart"
+                    >
+                      {vis && rows.length !== 0 && (
+                        <div className="dscHistogram" data-test-subj="discoverChart">
+                          <DiscoverHistogram
+                            chartData={histogramData}
+                            timefilterUpdateHandler={timefilterUpdateHandler}
+                          />
+                        </div>
+                      )}
+                    </section>
+                  )}
+
+                  <div className="dscResults">
+                    <section
+                      className="dscTable fixed-scroll"
```

Comment:
I would highly suggest a stronger scoped class than `fixed-scroll`. In fact you can just apply the style directly to `dscTable`.

Reply:
thx & done